web-ui/src/pages/PulseReportPage.jsx

```diff
@@ -535,6 +640,31 @@ const PulseReportPage = () => {
         </ResponsiveContainer>
       </CardContent>
     </Card>
+    <Card>
+      <CardHeader
+        title="Total Responses"
+        titleTypographyProps={{ variant: 'h5', component: 'h2' }}
+      />
+      <CardContent>
+        <ResponsiveContainer width="100%" aspect={3.0}>
+          <PieChart width={300} height={300}>
+          <Tooltip
+            wrapperStyle={{ color: "black", backgroundColor: "white", paddingLeft: "10px", paddingRight: "10px" }}
+            content={<CustomPieTooltip />}
+          />
+            <Pie
+              data={pieChartData}
+              dataKey="value"
+              nameKey="name"
+              fill={ociLightBlue}
```

Comment:
@ocielliottc Could you color these the same as they are on the bar chart (`color` from `dataInfo`)?

See: line 43-45 on https://recharts.org/en-US/examples/PieChartWithCustomizedLabel

Reply:
@mkimberlin I can.  Should we use shades of the blue or orange or the light blue?

Reply:
Why don't we do two charts? Blue for at work, orange for outside work. Show both if "Both" is selected in the toggle. Then it would feel like it worked similarly to the bar cart.

Reply:
That works.